renderer/src/shared/Hint/types.ts

```diff
@@ -0,0 +1,32 @@
+import { ReactNode, MutableRefObject } from "react";
+
+export type Position =
+  | {
+      basePosition: "bottom";
```

Comment:
You could use constant PLACEMENT here and replace the bottom, top, right, and left.